filebeat/docs/reference/configuration/filebeat-options.asciidoc

```diff
@@ -161,104 +161,109 @@ names added by Filebeat, then the custom fields overwrite the other fields.
+If this option is enabled, Filebeat ignores any files that were modified before the specified timespan. Configuring `ignore_older` can be especially useful if you keep log files for a long time. For example, if you want to start Filebeat, but only want to send the newest files and files from last week, you can configure this option.
 
-You can use time strings like 2h (2 hours) and 5m (5 minutes). The default is 0, which means disable.
+You can use time strings like 2h (2 hours) and 5m (5 minutes). The default is 0, which disables the setting.
 Commenting out the config has the same effect as setting it to 0.
 
-There are two different cases for files which fall under ignore_older:
+The files affected by this setting fall into two categories:
 
-* Files which were never harvested
-* Files which were harvested but weren't updated for longer then ignore_older
+* Files that were never harvested
+* Files that were harvested but weren't updated for longer than `ignore_older`
 
-In case a file was never harvested before and is updated, the reading will start from the beginning of the file as no state was persisted so far. For files which were harvested perviously, the state still exists and in case of an update, they will be continued at the last position.
+When a file that has never been harvested is updated, the reading starts from the beginning of the file because no state has been persisted. For a file that has been harvested previously, the state still exists, so when the file is updated, reading continues at the last position.
 
-For comparison, `ignore_older` relies on the modification time of the file. In case the modification time of files is not updated when its written to a file (can happen on Windows), `ignore_older` will start to ignore the file even though it could be that content was added at a later time.
+The `ignore_older` setting relies on the modification time of the file to determine if a file is ignored. If the modification time of the file is not updated when lines are written to a file (which can happen on Windows), the `ignore_older` setting may cause Filebeat to ignore files even though content was added at a later time.
 
-`ignore_older` can be especially useful if you keep log files for a long time and you start Filebeat, but only want to send the newest files to Elasticsearch and the old files from the last week, but not all files.
+To remove the state of previously harvested files from the registry file, use the `clean_inactive` configuration option.
 
-To remove the state from the registry file for files which were harvested before, the `clean_inactive` configuration option has to be used.
+Before a file can be ignored by the prospector, it must be closed. To ensure a file is no longer being harvested when it is ignored, you must set `ignore_older` to a longer duration than `close_inactive`. 
 
-
-Requirement: ignore_older > close_inactive
```

Comment:
It seemed redundant to me because you repeated it in the text. I thought maybe it was a note that you forgot to remove. Also, the angle bracket is a bit ambiguous. I realize it means "greater than", but users could potentially misinterpret this (for example, someone might think  it means "set this to"). Gotta account for all the ways that people can misinterpret things. :-) When we want to call something out as important, we use "important" notes. I'll flag this this one.